src/Microsoft.DotNet.Interactive.SqlServer/KustoConnectionDetails.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.DotNet.Interactive.SqlServer
+{
+    public class KustoConnectionDetails
```

Comment:
Internal DTO for passing values between what user provides and what is sent to the ToolsService